magento/magento2
# Fixed issue #18337

### Description (*)

**Search input is missing required attribute aria-expanded:** For this issue I have made changes in mediaCheck exit function because the exit function is using only for desktop vestion and it was using isExpandable=>false.   

### Fixed Issues (if relevant)

1. magento/magento2#18337: Search input is missing required attribute aria-expanded.
2. ...

### Manual testing scenarios (*)

1. Check home page in W3C html validator or open home page check with inspect element 
2. ...

### Contribution checklist (*)
